_includes/docs/2.5/database-setup.html

```diff
@@ -43,7 +43,7 @@ <h2 id="database-setup">Database Setup</h2>
 </pre>
 
 <h3>3. Creating the Azkaban Tables</h3>
-<p>Download the azkaban-sql-script tarball from the <a href="###">download page</a>. Contained in this archive are table creation scripts.</p>
+<p>Download the azkaban-sql-script tarball from the <a href='../../downloads.html'>download page</a>. Contained in this archive are table creation scripts.</p>
```

Comment:
That change is in keeping with the other links defined. That said I'll make the necessary change.